### ISSUE TITLE
NullpointerException when using the inferredConstCheck warning group

### ISSUE THREAD

**Issue by DrRataplan** [google/closure-compiler#2617]
During the upgrade to the new closure compiler version, I ran into the following error:

```
> java -jar compiler.jar --js "tmp.js" --jscomp_error=inferredConstCheck
java.lang.NullPointerException: No warning class for name: inferredConstCheck
        at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:864)
        at com.google.javascript.jscomp.DiagnosticGroups.setWarningLevel(DiagnosticGroups.java:733)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.setWarningGuardOptions(AbstractCommandLineRunner.java:322)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.setRunOptions(AbstractCommandLineRunner.java:337)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.doRun(AbstractCommandLineRunner.java:1046)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.run(AbstractCommandLineRunner.java:500)
        at com.google.javascript.jscomp.CommandLineRunner.main(CommandLineRunner.java:1958)
```

This worked in the 20170626 version. Is this flag deprecated? If so, could the documentation be updated at https://github.com/google/closure-compiler/wiki/Warnings?

**Comment by MatrixFrog** [comment]
This was actually removed in r20160315, so passing `--jscomp_error=inferredConstCheck` was a no-op. Then in d0cfcc1b4441b4e75c1c3a0f34720176e40580b1 we finally removed them. You can just remove `--jscomp_error=inferredConstCheck` from your flags.

**Comment by b-strauss** [comment]
FYI: it's still listed here

https://github.com/google/closure-compiler/wiki/Warnings
